src/views/primary-auth/SocialAuth.js

```diff
@@ -69,7 +64,7 @@ define([
           'data-se': attr
         },
         className: 'social-auth-button ' + attr,
-        title: socialAuthIdpLabels[type],
+        title: Okta.loc('socialauth.' + type + '.label'),
```

Comment:
It's already being evaluated after the view has been initialized - the only unsupported pattern is "Okta.loc is evaluated when the module itself is loaded".